packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx

```diff
@@ -46,6 +46,7 @@ const activeStyle = ({ theme: { textField }, variant }: InnerWrapperProps) => cs
         background-color: ${textField[variant].active.bgColor};
         &::after {
             border-color: ${textField[variant].active.borderColor};
+            border-radius: ${variant === 'outlined' || variant === 'fusion' ? textField[variant].active.borderRadius: '0'};
```

Comment:
Can we please write it like this -> ${variant === 'filled' ? '0' : textField[variant].active.borderRadius};.

Reply:
Yes, that's cleaner. Thank you!